README.md

```diff
@@ -16,7 +16,7 @@
 
 ## What is Regular Expression?
 
-> Regular expression is a group of characters or symbols which is used to find a specific pattern from a text.
+> Regular expression is a group of characters or symbols which are used to find a specific pattern from a text.
```

Comment:
I think that this changed is wrong...

For me is correct  way is: 

`Regular expression is a group of characters or symbols which is used to find a specific pattern from a text.`

Because the group (just one group) is used to find a specific pattern

Reply:
I agree with @eamanu.

TBH though, you could probably drop the `which (is|are)` part from that sentence and it'd still read in the same way and be perfectly acceptable anyhow:

`A regular expression is a group of characters or symbols used to find a specific pattern from a text.`

Reply:
Thanks but we will stick with `which is` because it fits more in our context